src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -118,6 +120,12 @@
     @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
     public int SAMPLE_SIZE=10000;
 
+    @Option(doc = "If true, fast algorithm is used.")
+    public boolean USE_FAST_ALGORITHM = false;
+
+    @Option(doc = "Average read length in the file. Default is 150.", optional = true)
```

Comment:
This interface was used during earlier stages of development. Code structure was changed and we don't need it anymore. We've removed it. Thank you for pointing it out.
If READ_LENGTH is much smaller than actual read length program will crash with ArrayIndexOutOfBoundException trying to increment value corresponding reference sequence index in Counter. We`ve added this possible cause of crashing into exception message.